config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :videos, only: [:index, :show, :create]
+  resources :customers, only: [:index]
+  resources :rentals, only: [:create, :destroy]
```

Comment:
You didn't need these routes since you only interact with rentals using the custom routes below.